application/classes/Ushahidi/Console/ProcessExportJobs.php

```diff
@@ -0,0 +1,131 @@
+<?php defined('SYSPATH') or die('No direct script access');
+
+/**
+ * Ushahidi Data Provider Console Commands
+ *
+ * @author     Ushahidi Team <team@ushahidi.com>
+ * @package    Ushahidi\Console
+ * @copyright  2018 Ushahidi
+ * @license    https://www.gnu.org/licenses/agpl-3.0.html GNU Affero General Public License Version 3 (AGPL3)
+ */
+
+use Ushahidi\Console\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Input\ArrayInput;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Output\BufferedOutput;
+use Ushahidi\Core\Entity\ExportJobRepository;
+
+/* Simple console command that processes pending jobs in the DB */
+
+class Ushahidi_Console_ProcessExportJobs extends Command {
+
+    private $data;
+    private $exportJobRepository;
+
+
+    public function setExportJobRepo(ExportJobRepository $repo)
+    {
+        $this->exportJobRepository = $repo;
+    }
+
+	protected function configure()
+	{
+		$this
+			->setName('processexports')
+			->setDescription('Processes pending export jobs.')
+			->addArgument('action', InputArgument::OPTIONAL, 'list, pending', 'pending')
+			;
+	}
+
+   //display all jobs, including pending, failed, and successful
+   protected function executeList(InputInterface $input, OutputInterface $output)
```

Comment:
If we really need a list command - add it as a separate class and command called `export:list` (make this one `exports:run` or something)